### DISCUSSION TITLE
Dropdown placeholder / prompt / blank items?

### DISCUSSION THREAD

**Discussion by jasper502** [orchidjs/tom-select#382] [Q&A]
I have a ajax populated select configured like this (load omitted for clarity):

```
plugins: ['dropdown_input'],
allowEmptyOption: true,
maxItems: 1,
valueField: 'value',
labelField: 'label',
searchField: 'label',
optgroupField: 'optgroup_id',
preload: true,
placeholder: "Search...",
closeAfterSelect: false,
```

With this HTML:

```
<select data-selected="" data-controller="ts-select" data-type="asset_category" class="select required tomselected ts-hidden-accessible" name="asset[asset_category_id]" id="asset_asset_category_id" tabindex="-1">
  <option value="">Select Category...</option>
  <option value=""></option>
</select>
```
and I get this:

<img width="362" alt="image" src="https://user-images.githubusercontent.com/1505829/168885882-8427eaf3-ea8f-4c53-aea7-35766801bcde.png">

with the prompt as a selected item and I am looking for this:

<img width="362" alt="image" src="https://user-images.githubusercontent.com/1505829/168886809-31695457-9e8c-4f06-986e-7d8695f98398.png">

I have tried all combinations of the HTML and TS placeholder / prompt / bank settings.

**Comment by oyejorge** [comment]
Try
```js
allowEmptyOption: false,
```
with
```html
<select data-selected="" data-controller="ts-select" data-type="asset_category" class="select required tomselected ts-hidden-accessible" name="asset[asset_category_id]" id="asset_asset_category_id" tabindex="-1">
  <option value="">Select Category...</option>
</select>
```

**Reply by jasper502** [reply]
That solves the dropdown state but now the select is empty and the prompt / placeholder is gone:

![image](https://user-images.githubusercontent.com/1505829/168956383-64ddea30-4662-46fb-82ab-c1abd7c04778.png)

**Comment by jasper502** [comment]
Less than ideal but works:

```
#asset_asset_category_id-opt-1 {

  display: none;

}
```
